package.json

```diff
@@ -13,8 +13,13 @@
     "@types/node": "^16.7.13",
     "@types/react": "^17.0.20",
     "@types/react-dom": "^17.0.9",
+    "i18next": "^21.6.14",
+    "i18next-browser-languagedetector": "^6.1.4",
+    "i18next-http-backend": "^1.4.0",
+    "luxon": "^2.3.1",
```

Comment:
Jeśli chodzi o formatowanie czasu to zobacz tą bibliotekę https://momentjs.com/
według mnie ma trochę przyjemniejszą dokumentację niż luxon

Reply:
@TenGosc007 Moment.js nie jest już zalecany. Biblioteka jest znana ale sami twórcy już odradzają korzystanie z niej:
![image](https://user-images.githubusercontent.com/84628957/160477869-b6396ac9-a853-4405-a52c-dc3994e56db5.png)

Reply:
pewnie, może zostać Luxon, mi po prostu dokumentacja momentJS pasuje i tyle, ale nie jest to najlżejsza biblioteka